src/index.js

```diff
@@ -3,17 +3,20 @@ const H5P = require('h5p-nodejs-library');
 const defaultEditorIntegration = require('./default_editor_integration');
 const defaultTranslation = require('./translations/en.json');
 const defaultRenderer = require('./renderers/default');
-const defaultContentTypeCache = require('./content_type_cache');
+
+const ContentTypeCache = require('../src/content-type-cache');
+const ContentTypeInformationRepository = require('../src/content-type-information-repository');
 
 class H5PEditor {
-    constructor(storage, baseUrl = '/h5p', ajaxPath = '/ajaxPath?action=') {
+    constructor(storage, baseUrl = '/h5p', ajaxPath = '/ajaxPath?action=', storage2, config, libraryManager, user) {
```

Comment:
I think we should rename `storage2` to something like `valueStorage` as `storage2` is not very descriptive.

Reply:
I've renamed it to keyValueStorage